packages/module-source/test/fixtures/unmangled.js

```diff
@@ -0,0 +1,10 @@
+({   imports: $h͏_imports,   liveVar: $h͏_live,   onceVar: $h͏_once,   importMeta: $h͏____meta, }) => (function () { 'use strict';   $h͏_imports([]);Object.defineProperty(createBinop, 'name', {value: "createBinop"});$h͏_once.createBinop(createBinop);   // deliberately offset
```

Comment:
Where is the closing paren for the function expression here?

Reply:
Mmm it should be at then end of the file right? It looks like I forgot to flush some tokens somewhere 🤔

Reply:
Actually, cloning the PR locally I get this additional line at the end of the file:
```js
})()
```

Probably @kriskowal accidentally deleted it.

Reply:
Pushed the fix for this. It was present locally when I got the test to pass.